### PR TITLE
1.3.6

### PR DESCRIPTION
**Pull request by ozsay**

### CODE REVIEW

File: packages/angular-meteor-data/modules/core.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+angular.module('angular-meteor.core', [
+  'angular-meteor.utils',
```
Comment (kamilkisiela) [inline packages/angular-meteor-data/modules/core.js]: @ozsay Shouldn't be here `angular-meteor.utilities`?

File: packages/angular-meteor-data/modules/view-model.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+angular.module('angular-meteor.view-model', [
+  'angular-meteor.utils',
```
Comment (kamilkisiela) [inline packages/angular-meteor-data/modules/view-model.js]: Same as here
https://github.com/Urigo/angular-meteor/pull/1216/files#r52520790

Comment (ozsay) [inline packages/angular-meteor-data/modules/view-model.js]: Yeah it should

Thanks

Comment (kamilkisiela) [inline packages/angular-meteor-data/modules/view-model.js]: I'm going through angular-meteor line by line so it came up. Good job @ozsay with 1.3.6! :)

**Pull request merged**